### ISSUE TITLE
The issue of the input data shape

### ISSUE THREAD

**Issue by dongqunxi** [scot-dev/scot#179]
Dear @mbillingr and @cbrnr ,

I make a comparison about the input data shape between before and after SCot versions.
Take the ```scot.var.VAR.fit()``` as an example,
Old version, the description of the data shape is:
```ruby
def fit(self, data):
        """ Fit VAR model to data.
        
        Parameters
        ----------
        data : array-like
            shape = [n_samples, n_channels, n_trials] or
            [n_samples, n_channels]
            Continuous or segmented data set.
       ...
       """
```
Current version, the description of the data shape is:
```ruby
 def fit(self, data):
        """Fit VAR model to data.
        
        Parameters
        ----------
        data : array, shape (trials, channels, samples) or (channels, samples)
            Epoched or continuous data set.
            
        ...
        """
```
The shape of the input data changes, can you explain the reason why the shape is changed?

Some functions in your current version, you apply a different data shape, such as in the function
``` optimize_order``` , the data shape is ```(n_trials, n_samples, n_channels)```. I wonder why shouldn't unify the same data shape? 
Hope to get your feedback. Thanks a lot.

**Comment by cbrnr** [comment]
We decided to change the shape to (epochs, channels, samples) because first this is what MNE Python uses, and second, this is also more convenient when using functions from `numpy` or `scipy`, because these work on the last axis by default - and usually, we want to apply functions over the samples.

Thanks for pointing out this inconsistency in `optimize_order` - I guess we've overlooked that in our refactoring. @mbillingr, I guess this is not intentional and we should open an issue?
